src/net.h

```diff
@@ -64,6 +64,8 @@ static const unsigned int MAX_PROTOCOL_MESSAGE_LENGTH = 4 * 1000 * 1000;
 static const unsigned int MAX_SUBVERSION_LENGTH = 256;
 /** Maximum number of automatic outgoing nodes over which we'll relay everything (blocks, tx, addrs, etc) */
 static const int MAX_OUTBOUND_FULL_RELAY_CONNECTIONS = 8;
+/** Maximum number of automatic fullrbf peers */
+static const int MAX_FULLRBF_RELAY_CONNECTIONS = 4;
```

Comment:
To be clear, the idea is to have 4 additional full-rbf outbound connections, in addition to the 8 full-outbound and 2 block-relay-only connections, correct?
I think this isn't currently working as intended, because `m_max_outbound` (which sets the number of semaphores for outbound connections) is not adjusted in `CConnman::Init()`, so we'd currently make just 1 full-rbf connection (and stop making feelers completely).

Reply:
rearranging the lines and accumulating the proper value results in 4 connections

Reply:
> To be clear, the idea is to have 4 additional full-rbf outbound connections, in addition to the 8 full-outbound and 2 block-relay-only connections, correct?

Right, the idea is to have 4 additional full-rbf outbound connections, in addition to the 8 full-outbound and 2 blocks-relay-only connections. Modified `m_max_outbound_fullrbf_relay` in `init.cpp` to account for `nMaxConnections` and number of full-outbound and block-relay-only connections, and modified `m_max_outbound` computation in `CConman::Init()`. The feelers 1 slot connection should be preserved.

Thanks for the catchup.

Reply:
> > To be clear, the idea is to have 4 additional full-rbf outbound connections, in addition to the 8 full-outbound and 2 block-relay-only connections, correct?
> 
> Right, the idea is to have 4 additional full-rbf outbound connections, in addition to the 8 full-outbound and 2 blocks-relay-only connections.

The second commit title and message body still needs to be updated (as would docs like, offhand, reduce-memory.md and reduce-traffic.md and perhaps elsewhere), and other affected p2p logic and behavior reviewed carefully.

Reply:
Addressed, modifying reduce-memory.md, reduce-traffic.md to notify the change in number of outbound connections.